modules/system-management/pages/workload-management.adoc

```diff
@@ -267,7 +267,7 @@ You must use separate commands for each entity type.
 ----
 GET gsql/v1/workload-manager/permission
 ----
-Show info on a specific workload queue or all.
+Lists information about all or specific workload queues, including permissions and details.
```

Comment:
@Tushar-TG-14 It sounds like you are describing the SHOW command, not get.
What is the actual intent of this example? Should the keyword above be changed to SHOW?